share/github-backup-utils/ghe-backup-store-version

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+#/ Usage: ghe-backup-store-version
+#/ Stores information about the used version of backup-utils on <host>
+set -e
+
+# Bring in the backup configuration
+cd $(dirname "$0")/../..
+. share/github-backup-utils/ghe-backup-config
+
+# Perform a host-check and establish GHE_REMOTE_XXX variables.
+ghe_remote_version_required "$host"
```

Comment:
`$host` is going to be empty here, needs `$GHE_HOSTNAME`. No arguments will work just fine also, `ghe-host-check` (used by `ghe_remote_version_required`) uses `$GHE_HOSTNAME` if no arguments are provided.